saidump/saidump.cpp

```diff
@@ -199,6 +429,13 @@ int main(int argc, char ** argv)
         }
     }
 
+    if (g_cmdOptions.dumpGraph)
+    {
+        dumpGraph(dump);
+
+        return EXIT_SUCCESS;
```

Comment:
>EXIT_SUCCESS [](start = 15, length = 12)

How about no exit, and continue to dump everything?

Reply:
no, since graph is generated on the stdout so it could be dumped to *.gv file, and if other stuff would be generated then it would mess output